lib/editor/components/pattern/PatternStopsPanel.js

```diff
@@ -128,15 +129,17 @@ export default class PatternStopsPanel extends Component<Props, State> {
           {`Stops & Locations (${patternHaltCount})`}
         </h4>
         <div style={{width: '100%'}}>
-          <Button
-            onClick={this._clickNormalizeStopTimes}
-            style={{marginBottom: '5px'}}
-            className='pull-right'
-            block
-            bsSize='small'>
-            <Icon type='clock-o' />{' '}
-            Normalize stop times
-          </Button>
+          {!patternHasLocations &&
```

Comment:
I'm not sure if I prefer disabling to fully removing the button in this case, I'll defer to the second reviewer.

Reply:
Let's get rid of the button!